docs/md_docs/flux_examples.md

```diff
@@ -0,0 +1,273 @@
+```
+
+## Make the model
+
+Now let's make the model we will use to fit the data. First, let's make the spectral function for source_1 and set priors on the parameters.
+
+
+```python
+spectrum_1 = Blackbody()
+
+spectrum_1.K.prior = Log_normal(mu=np.log(1e-7), sigma=1)
+spectrum_1.kT.prior = Log_normal(mu=np.log(300), sigma=2)
+
+ps1 = PointSource("src1", ra=1, dec =20, spectral_shape=spectrum_1)
+
+```
+
+We will do the same for the other source but also include the power law component 
+
+```python
+spectrum_2 = Blackbody() + Powerlaw_flux(a=10, b=500) # a,b are the bounds for the flux for this model
+
+spectrum_2.K_1.prior = Log_normal(mu=np.log(1e-6), sigma=1)
+spectrum_2.kT_1.prior = Log_normal(mu=np.log(300), sigma=2)
+
+spectrum_2.F_2.prior = Log_normal(mu=np.log(1e2), sigma=1)
+spectrum_2.F_2.bounds = (None, None)
+
+spectrum_2.index_2.prior = Gaussian(mu=-1., sigma = 1)
+spectrum_2.index_2.bounds = (None, None)
+
+ps2 = PointSource("src2", ra=2, dec=-10, spectral_shape=spectrum_2)
+```
+
+Now we can combine these two sources into our model.
+
+```python
+model = Model(ps1, ps2)
+```
+
+### Linking parameters
+
+We hypothesized that both sources should have the a same blackbody temperature. We can impose this by linking the temperatures. 
+
+
+```python
+model.link(model.src1.spectrum.main.Blackbody.kT, 
+           model.src2.spectrum.main.composite.kT_1)
+```
+
+we could also link the parameters with an arbitrary function rather than directly. Check out the [astromodels documentation](https://astromodels.readthedocs.io/en/latest/Model_tutorial.html#linking-parameters) for more details.
+
+```python
+model
+```
+
+### Assigning sources to plugins
+
+Now, if we simply passed out model to the BayesianAnalysis or JointLikelihood objects, it would sum the point source spectra together and apply both sources to all data. 
+
+This is not what we want. Many plugins have the ability to be assigned directly to a source. Let's do that here:
+
+```python
+spectrum_generator_1.assign_to_source("src1")
+
+spectrum_generator_2.assign_to_source("src2")
+```
+
+Now we simply make our our data list
+
+```python
+data = DataList(spectrum_generator_1, spectrum_generator_2)
+```
+
+## Fitting the data
+
+Now we fit the data as we normally would. We use Bayesina analysis here.
+
+```python
+ba = BayesianAnalysis(model, data)
+ba.set_sampler("ultranest")
+ba.sampler.setup(frac_remain=0.5)
+_ = ba.sample()
+```
+
+Let's examine the fits.
+
+```python
+fig = display_spectrum_model_counts(ba);
+ax = fig.get_axes()[0]
+ax.set_ylim(1e-6)
+```
+
+Lets grab the result. Rememer, we can save the results to disk, so all of the following ooperations can be run at a later time without having to redo all the above steps!
+
+```python
+result = ba.results
+result.corner_plot();
+```
+
+## Computing fluxes
+
+Now we will compute fluxes. We can compute them an many different units, over any energy range also specified in any units. 
```

Comment:
Suggestion: Mention explicitly that we can calculate photon fluxes (in units of photons / time/ area), energy fluxes (energy/time/area) or nuFnu fluxes ( energy^2 / time /area [????] ), and the user can pick arbitrary units for energy/area/time in each case.